clickhouse-25.2.yaml

```diff
@@ -169,6 +169,15 @@ subpackages:
           ln -s /dev/stderr ${{targets.contextdir}}/var/log/clickhouse-server/clickhouse_error.log
           mkdir -p ${{targets.contextdir}}/var/lib
           ln -s /bitnami/clickhouse/data ${{targets.contextdir}}/var/lib/clickhouse
+    test:
+      environment:
+        contents:
+          packages:
+            - ${{package.name}}
+      pipeline:
+        - uses: bitnami/validate-welcome-message
```

Comment:
Is this missing an epoch bump?

Reply:
Yep! Resolved a few merge conflicts and in the process, reset the epoch bump. I'll get on that, thank-you for spotting and calling out